cmd/utils/nodecmd/snapshot.go

```diff
@@ -109,6 +123,13 @@ reading all nodes and logging the missing nodes.
 				utils.LevelDBCompressionTypeFlag,
 				utils.DataDirFlag,
 				utils.ChainDataDirFlag,
+				utils.RocksDBSecondaryFlag,
+				utils.RocksDBCacheSizeFlag,
+				utils.RocksDBDumpMallocStatFlag,
+				utils.RocksDBFilterPolicyFlag,
+				utils.RocksDBCompressionTypeFlag,
+				utils.RocksDBBottommostCompressionTypeFlag,
+				utils.RocksDBDisableMetricsFlag,
```

Comment:
Could the flags be different for each command? If it isn't, It would be better to make a global variable like "snapshotFlags"

Reply:
@kjeom Agreed. Instead of `snapshotFlags`, how about using `DBFlags`? There is already `DBMigrationSrcFlags` including the same flags related to database. So, renaming `DBMigrationSrcFlags` to `DBFlags` is also necessary. What do you think?

Reply:
Hmm.. DBMigrationSrcFlags looks a little bit different with DBMigrationDstFlags. Can we make DB related all the things DBFlags? I think it's our best.

Reply:
@kjeom Wait. It is not the same. I introduced snapshot flag group. seems a bit duplicated, but it looks more clear.